core/mingw/src/files/FileSystemMingw.kt

```diff
@@ -38,6 +38,11 @@ internal actual fun basenameImpl(path: String): String {
 
 internal actual fun isAbsoluteImpl(path: String): Boolean {
     if (path.startsWith(SystemPathSeparator)) return true
+    if (path.length > 1 && path[1] == ':') {
+        if (path.length == 1) return false
```

Comment:
`path.length > 1` and inside `path.length == 1` ?!

Reply:
Seems like I need more tests